flake.nix

```diff
@@ -70,9 +69,9 @@
       # artifacts from above.
       alpacka = craneLib.buildPackage {
         inherit cargoArtifacts src buildInputs;
+        doCheck = false;
```

Comment:
Not sure why we want to disable tests?

Reply:
Don't rerun tests since we already have nexttest setup